ee-9/feature-pack/src/main/resources/license/preview-feature-pack-licenses.xml

```diff
@@ -184,7 +189,7 @@
           <distribution>repo</distribution>
```

Comment:
Is EPL correct?  Unless I am in the wrong repo is it CDDL?

Reply:
I think that's an older repo. We're using version 2.0.1.Final, which comes from https://github.com/jboss/jboss-jakarta-annotations-api_spec